packages/app/src/app/pages/Sandbox/Editor/Header/types.ts

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export interface HeaderProps extends React.FunctionComponent {
```

Comment:
@SaraVieira U can just use `React.FC` instead of `React.FunctionComponent`

```suggestion
export interface HeaderProps extends React.FC {
```

Reply:
I am gonna keep it this way because I find the way people use TS with only two letters super confuysing and this way I can tell what it is :(

Ping me on slack if you don't agree